app/ide-desktop/lib/content/src/panic.tsx

```diff
@@ -0,0 +1,87 @@
+/** @file This file defines a component that is responsible for displaying a user-facing message
+ * about an application crash (panic). The component is used as a body of a Toastify toast. */
+
+import * as React from 'react'
+import * as toastify from 'react-toastify'
+
+/** Props for `PanicMessage` component. */
+export interface PanicMessageProps {
+    /** The panic message with stack trace. Usually big multiline text. */
+    message: string
+    /** A callback to trigger application restart. */
+    restart: () => void
+}
+
+/** A component displaying panic message inside a toast. */
+export function PanicMessage(props: PanicMessageProps) {
+    const reportUrl =
+        'https://github.com/enso-org/enso/issues/new' +
+        '?labels=--bug%2Ctriage' +
+        '&template=bug-report.yml'
+    return (
+        <div className="flex flex-col">
+            <h1 className="text-xl font-semibold">Enso has crashed.</h1>
+            <p className="mt-3">
+                Enso has encountered a critical error and needs to be restarted. This is a bug, and
+                we would appreciate it if you could report it to us.
+            </p>
+            <p>Please include following panic message in your report:</p>
+            <pre
+                className={
+                    'overflow-auto mt-2 p-2 bg-gray-200 text-gray-800 border rounded ' +
```

Comment:
i've been keeping these as a single line (which `eslint` is fine with)
i think it's fine, because `className="whatever"` does not contain any logic anyway, so there is no harm in letting it go past the right edge of the screen